src/components/Editor/index.js

```diff
@@ -253,6 +253,14 @@ class Editor extends PureComponent<Props, State> {
     if (selectedSource && isLoaded(selectedSource.toJS())) {
       this.highlightLine();
     }
+
+    // NOTE: when devtools are opened, the editor is not set when
+    // the source loads so we need to wait until the editor is
+    // set to update the text and size.
+    if (!prevState.editor && this.state.editor) {
```

Comment:
Only note if you're looking for existence it maybe better to do this as sometimes you may get null or undefined.
```js
if (prevState.editor == null && this.state.editor != null)`
```